test/index.test.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path')
+
+const test = require('tape')
+const {
+  extensions,
+  languages,
+  Uri,
+  window,
+  workspace,
+  Position
+} = require('vscode')
+
+/**
+ * Wait for a given amount of time.
+ *
+ * @param {number} ms How long to sleep in milliseconds
+ */
+function sleep(ms) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms)
+  })
+}
+
+/**
+ * @returns {Promise<void>}
+ */
+module.exports.run = () =>
```

Comment:
The test file must be loadable by `@vscode/test-electron`, meaning it has to use CJS.